CHANGELOG.md

```diff
@@ -2,6 +2,10 @@ Changelog for the gruf gem. This includes internal history before the gem was ma
 
 ### Pending release
 
+### 2.5.2
```

Comment:
Do we need to bump version file as well ? or we have more PR's being worked on as part of this release.

Reply:
Dropped that changelog header for now, that will come in separate commit, as per https://github.com/bigcommerce/gruf/pull/70#issuecomment-464858867